<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solarus Quest Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QuestTreeView" name="quest_tree_view"/>
      <widget class="QTreeView" name="tmpTreeView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuest">
    <property name="title">
     <string>Quest</string>
    </property>
    <addaction name="actionNew_quest"/>
    <addaction name="actionLoad_quest"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_quest"/>
   </widget>
   <addaction name="menuQuest"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_quest">
   <property name="text">
    <string>New quest...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad_quest">
   <property name="text">
    <string>Load quest...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRun_quest">
   <property name="text">
    <string>Run quest</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QuestTreeView</class>
   <extends>QTreeView</extends>
   <header>quest_tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
